### PR TITLE
doc(v1.27.6-rev.1): update compatibility matrix and release note

### PR DESCRIPTION
**Pull request by g-iannelli**
- [x] Update main README
- [x] Create Release Note
- [x] Update Compatibility Matrix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,9 +45,9 @@ Check the [compatibility matrix][compatibility-matrix] for additional informatio
...
 
-| Tool                    | Version   | Description                                                                                                                                                |
-| ----------------------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [furyctl][furyctl-repo] | `>=0.25.0` | The recommended tool to download and manage KFD modules and their packages. To learn more about `furyctl` read the [official documentation][furyctl-repo]. |
+| Tool                    | Version    | Description                                                                                                                                                |
+| ----------------------- |------------| ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [furyctl][furyctl-repo] | `>=0.27.0` | The recommended tool to download and manage KFD modules and their packages. To learn more about `furyctl` read the [official documentation][furyctl-repo]. |
```
Comment (ralgozino) [inline README.md:50]: shouldn't this be 0.27.2?
```suggestion
| [furyctl][furyctl-repo] | `>=0.27.2` | The recommended tool to download and manage KFD modules and their packages. To learn more about `furyctl` read the [official documentation][furyctl-repo]. |
```

**Review by nutellinoit — CHANGES_REQUESTED** [review]
Overall LGTM, with some changes:
- Missing badge update on the top of the README.md
- Also, is there some example configurations on how to configure containerd credentials somewhere?

File: docs/releases/v1.27.6-rev.1.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# On Premises add-on module release 1.27.6-rev.1
+
+Welcome to the latest release of `on-premises` module of [`Kubernetes Fury Distribution`](https://github.com/sighupio/fury-distribution) maintained by SIGHUP team.
+
+This minor release is compatible with Kubernetes 1.27.6 and add Nvidia and registries configuration support to ContainerD.
```
Comment (ralgozino) [inline docs/releases/v1.27.6-rev.1.md:5]:
```suggestion
This revision release is compatible with Kubernetes 1.27.6 and adds Nvidia container toolkit and registries configuration support to ContainerD.
```

**Review by ralgozino — CHANGES_REQUESTED** [review]
some minor suggestions

File: README.md (hunk 1 above)
Comment (g-iannelli) [reply README.md:50]: I don't know if last patch is required. Maybe it will work also with 0.27.0

Comment (ralgozino) [reply README.md]: you are right 👍  the containerd features don't need the features included in furyctl's latest patch

**Review by nutellinoit — APPROVED** [review]
LGTM

**Review by ralgozino — APPROVED** [review]
lgtm

**Pull request merged**